### ISSUE TITLE
Slow 60fps Videos on Kodi since Berryboot 2018.03.x

### ISSUE THREAD

**Issue by agoldcheidt** [maxnet/berryboot#496]
Hello everyone, I have a Raspberry Pi 2 B with LibreELEC and I started to noticed that videos (1080p at 60fps) like this one: https://www.youtube.com/watch?v=bsS2apzzdiA are playing slowly, unwatchable. I've downloaded that video via youtube-dl and I can see it amazingly with Berryboot: berryboot-20180216-pi2-pi3.zip. I don't put any extra line in the config.txt file, I use the default one. Anyone with the same issue? Possible fix?

**Comment by maxnet** [comment]
Your subject says Kodi, your post content says youtube-dl
What are you using exactly?

**Comment by agoldcheidt** [comment]
Hi @maxnet, yes, I download videos from YouTube with youtube-dl for offline viewing and watch them with LibreELEC and OpenELEC, both are based on Kodi. For some reason all videos at 60fps are playing very slowly in both distros with any Berryboot version since 2018.03.x.

**Comment by maxnet** [comment]
Tried other hardware accelerated video playing programs like `omxplayer` ?

**Comment by agoldcheidt** [comment]
Ok, I've searched about it and I've found this: https://forum.kodi.tv/showthread.php?tid=263052 so it looks like omxplayer is better than MMAL. I tested that option with berryboot-20180405-pi2-pi3.zip, deactivate the MMAL hardware acceleration and activate the OMXPLAYER hardware acceleration. Results: videos at 60fps runs better but still slow, the video run slower than the audio. I think this issue is more related to Kodi than Berryboot. I'll continue doing some tests, if I find a fix, of course I'll share it.
